docs/reference/adobe/experience/customerJourneyManagement/message-execution-timeline.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/message-execution-timeline",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - Message Execution Timeline",
+  "description": "Captures an array of time-profiling info, for every component of execution pipeline, corresponding to every message being executed.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "time-profiling": {
+      "type": "object",
+      "description": "This datatype is used to store the time profiling information of any service or component.",
+      "properties": {
+        "xdm:componentName": {
+          "title": "serviceName",
+          "description": "The name of the service/component, for which time information is captured.",
+          "type": "string"
+        },
+        "xdm:serviceGroup": {
+          "title": "serviceGroup",
+          "description": "An optional attribute, to capture the timeline of a service, at sub-tree level.",
+          "type": "string"
+        },
+        "xdm:startTime": {
+          "title": "startTime",
+          "description": "Execution Start time in Epoch millis for the component.",
+          "type": "number"
```

Comment:
I feel, we should be using the date-time format for specifying the start/end date. If we keep type as number, we won't get any automatic validation, whether some one has entered the valid date or not. Number means any negative integer also.
Anyway while querying the data we can convert the date-time format to epoch if required.

Also, I see this as a general practice at other places in XDM: https://github.com/adobe/xdm/blob/cf79e096f7f16270b133b0e8609d04b6faf7cb2f/docs/reference/adobe/experience/decisioning/activity-detail.schema.json